File: 20170518/KotlinInAndroid/app/src/main/java/com/plter/kotlininandroid/Hello.kt
package com.plter.kotlininandroid

import android.content.Context
import android.widget.Toast

/**
 * Created by plter on 5/18/17.
 */
class Hello {

    fun sayHello(context:Context){
        Toast.makeText(context,"Hello",Toast.LENGTH_SHORT).show();
    }
}

File: 20170616/HelloCPP/Hello.cpp
//
// Created by plter on 6/16/17.
//

#include "Hello.h"
#include <stdio.h>

void Hello::sayHello() {
    printf("Hello %s\n", name.c_str());
}

void Hello::sayHi() {
    printf("Hi\n");
}

Hello::Hello(const std::string &name) : name(name) {}

